### PR TITLE
[OKTA-702250] Feature Lifecycle Management page

### PR DESCRIPTION
**Pull request by annejuan-okta**
## Description:
- **What's changed?** Updated various parts of the feature lifecycle management concept page, improving the Acrolinx score

### Resolves:

* [OKTA-702250](https://oktainc.atlassian.net/browse/OKTA-702250)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md](https://github.com/okta/okta-developer-docs/blob/e00db93042e31ecf3a69734ed317c94e83f6ce3e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Ffeature-lifecycle-management%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7dc438e6-59d3-4745-a4ee-e56ccc1bd113) | 100  | 100  | 63  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b7dcd7da-03d9-4f25-a2a3-d0064456c78b?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md](https://github.com/okta/okta-developer-docs/blob/fd82baf7cb697ad1e1e5cd89d7742fd1b13a246f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Ffeature-lifecycle-management%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4d96731f-7c5d-405f-a0fd-251027523c62) | 100  | 100  | 63  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.c37d7f43-0bca-41e6-bb06-a90568149e13?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md](https://github.com/okta/okta-developer-docs/blob/412b731944602acad4ea6cfa7372db35a60e8683/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Ffeature-lifecycle-management%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/697e796c-faf0-4132-b547-94ac06c9fef4) | 100  | 100  | 63  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.2929fbbe-f786-479e-ab3f-526d7d8c86e9?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md](https://github.com/okta/okta-developer-docs/blob/304963914047043b74ea0e19cb5836c8d423d5d8/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Ffeature-lifecycle-management%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0f18d2d6-dcda-41a7-a62a-ec6db1ce25c5) | 100  | 100  | 63  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.909f8b81-3ec9-421f-9c4d-870c7f1d039f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md [hunk 1]
```diff
@@ -52,6 +54,6 @@ featA ..> featD
 In this example:
 
 * Feature A has three dependencies: Feature B, Feature C, and Feature D.
-* Feature D has one dependent, which is Feature A
+* Feature D has one dependant: Feature A.
 
-If Feature A has Feature B as a dependency, that means that Feature A cannot be enabled if Feature B is not enabled first. Conversely, Feature B has Feature A as one of its dependents. This means that Feature B cannot be disabled unless you disable Feature A first.
+If Feature A has Feature B as a dependency, then Feature A can't be enabled if Feature B isn't enabled first. Conversely, Feature B has Feature A as one of its dependants. This means that Feature B can't be disabled unless you disable Feature A first.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md:59]:
```suggestion
If Feature A has Feature B as a dependency, then you can't enable Feature A unless you enable Feature B first. Conversely, Feature B has Feature A as one of its dependants. This means that you can't disable Feature B unless you disable Feature A first.
```

File: packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md [hunk 2]
```diff
@@ -2,27 +2,29 @@
...
 
-## Self-service Features
+See [Release lifecycle](https://developer.okta.com/docs/api/openapi/okta-management/guides/release-lifecycle/).
 
-Self-service features are features that can be enabled or disabled by an Okta admin directly. Features that are not self-service can only be enabled or disabled by getting in touch with Okta Support. The [Okta Features API](/docs/reference/api/features/) is used to list, enable, and disable self-service features.
+## Self-service features
 
-> **Note:** A Beta feature that is marked as self-service will stay as self-service even if it's closed and temporarily unavailable to be enabled. For example, if an [open Beta](#beta-features) transitions into a closed Beta, the feature will remain self-service. This means that it will appear in the list of available self-service Features, but you will not be able to enable it without contacting Support.
+Self-service features are features an Okta admin can directly enable or disable. Features that aren't self-service can only be enabled or disabled by getting in touch with Okta Support. The [Okta Features API](/docs/reference/api/features/) is used to list, enable, and disable self-service features.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md:13]:
```suggestion
Self-service features are features that you can directly enable or disable. Contact [Okta Support](https://support.okta.com) to enable features that aren't self-service. The [Okta Features API](/docs/reference/api/features/) is used to list, enable, and disable self-service features.
```

**Review by susanharper-okta — COMMENTED** [review]
couple edits. LMK if you have questions and we can discuss.

File: packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md [hunk 3]
```diff
@@ -2,27 +2,29 @@
...
 
-Only Beta and Early Access (EA) features are considered self-service. Once a feature transitions from either of these states to Generally Available (GA), it will no longer be listed as an available self-service Feature.
+> **Note:** A Beta feature marked as self-service will stay as self-service, even if it's closed and temporarily unavailable to be enabled. For example, if an [open Beta](#beta-features) transitions into a closed Beta, the feature will remain self-service. This means that it appears in the list of available self-service features, but you can't enable it without contacting Support.
```
[the remark this answers is not in the document]
Comment (annejuan-okta) [reply packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md]: I actually went back and forth on whether this should be present or future tense. I'll try to keep this in mind for the future.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md](https://github.com/okta/okta-developer-docs/blob/15877519d9c0ff62cba4b3e344ac6a9bb42d2203/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Ffeature-lifecycle-management%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2f04a784-3d31-4847-8a02-ed6fd6254513) | 100  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.4a8e31dd-b257-42ed-82d8-a9ba62c873ee?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md [hunk 4]
```diff
@@ -2,27 +2,29 @@
 title: Feature Lifecycle Management
 ---
 
-# Feature Lifecycle Management
+# Feature lifecycle management
 
-This page will cover a few key concepts that underpin the [Okta Features API](/docs/reference/api/features/). The Features API allows you to manage self-service features, as well as their feature dependents and dependencies. Special restrictions apply to the behavior of the API with respect to Beta features.
+This page covers a few key concepts that underpin the [Okta Features API](/docs/reference/api/features/). The Features API allows you to manage self-service features, their feature dependants, and their feature dependencies. For Beta features, special restrictions apply to the behavior of the API.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md:7]:
```suggestion
This page covers a few key concepts that underpin the [Okta Features API](/docs/reference/api/features/). The Features API allows you to manage self-service features, their feature dependents, and their feature dependencies. For Beta features, special restrictions apply to the behavior of the API.
```

File: packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md [hunk 5]
```diff
@@ -52,6 +54,6 @@ featA ..> featD
 In this example:
 
 * Feature A has three dependencies: Feature B, Feature C, and Feature D.
-* Feature D has one dependent, which is Feature A
+* Feature D has one dependant: Feature A.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md:57]:
```suggestion
* Feature D has one dependent: Feature A.
```

File: packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md [hunk 6]
```diff
@@ -2,27 +2,29 @@
...
-> **Note:** Beta Features are only available in Preview cells.
+Though all available Beta features qualify as "self-service", Betas are either open or closed. You can enable or disable open Betas using the API. But, you can only disable closed Betas. If a feature has dependencies that are in closed Beta, then you can't enable it. If you want to enable a closed Beta, [contact Support](https://support.okta.com). When you enable an open Beta, it triggers an email to the admin who performed the action.
 
-## Dependencies and Dependents
+> **Note:** Beta features are only available in Preview orgs.
 
-Features may have dependencies and dependents. Dependencies are other features that the feature relies on, whereas dependents are features that rely on the feature.
+## Dependencies and dependants
+
+Features might have dependencies and dependants. Dependencies are other features that the feature relies on, whereas dependants are features that rely on the feature.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md:27]:
```suggestion
Features might have dependencies and dependents. Dependencies are other features that the feature relies on, whereas dependents are features that rely on the feature.
```

File: packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md [hunk 7]
```diff
@@ -52,6 +54,6 @@ featA ..> featD
 In this example:
 
 * Feature A has three dependencies: Feature B, Feature C, and Feature D.
-* Feature D has one dependent, which is Feature A
+* Feature D has one dependant: Feature A.
 
-If Feature A has Feature B as a dependency, that means that Feature A cannot be enabled if Feature B is not enabled first. Conversely, Feature B has Feature A as one of its dependents. This means that Feature B cannot be disabled unless you disable Feature A first.
+If Feature A has Feature B as a dependency, then you can't enable Feature A unless you enable Feature B first. Conversely, Feature B has Feature A as one of its dependants. This means that you can't disable Feature B unless you disable Feature A first.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md:59]:
```suggestion
If Feature A has Feature B as a dependency, then you can't enable Feature A unless you enable Feature B first. Conversely, Feature B has Feature A as one of its dependents. This means that you can't disable Feature B unless you disable Feature A first.
```

**Review by grahamsmith-okta — APPROVED** [review]
Recommend using the American spelling of "dependent" (instead of with the 'a')

Otherwise good to go

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md](https://github.com/okta/okta-developer-docs/blob/9657f7e21f215a3404e601f7e439b91b28c8c5db/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Ffeature-lifecycle-management%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a2000389-ab81-4358-bdd6-3c3fc814d346) | 100  | 45  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.3dc04159-588f-4763-8c6b-f5d7bbbc62a2?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md [hunk 8]
```diff
@@ -2,27 +2,29 @@
...
 
-Though all available Betas qualify as "self-service features", Betas can be either open or closed. Open Betas can be both enabled or disabled using the API, however closed Betas can only be disabled. If a feature has Feature dependencies that are in Closed Beta, then you will not be able to enable it. If you'd like to enable a closed Beta, you should [contact Support](https://support.okta.com). Enabling an Open Beta will trigger an email to the admin who performed the action.
+## Beta features
 
-> **Note:** Beta Features are only available in Preview cells.
+Though all available Beta features qualify as "self-service", Betas are either open or closed. You can enable or disable open Betas using the API. But, you can only disable closed Betas. If a feature has dependencies that are in closed Beta, then you can't enable it. If you want to enable a closed Beta, [contact Support](https://support.okta.com). When you enable an open Beta, it triggers an email to the admin who performed the action.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md:21]:
```suggestion
Though all available Beta features qualify as "self-service," Betas are either open or closed. You can enable or disable open Betas using the API. But, you can only disable closed Betas. If a feature has dependencies that are in closed Beta, then you can't enable it. If you want to enable a closed Beta, [contact Support](https://support.okta.com). When you enable an open Beta, it triggers an email to the admin who performed the action.
```

**Review by grahamsmith-okta — CHANGES_REQUESTED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md](https://github.com/okta/okta-developer-docs/blob/d33e3d16f6fc12e3dcebd1276ee843a2544b2fe8/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Ffeature-lifecycle-management%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3625a2f2-face-486d-8678-b511184a2102) | 100  | 45  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.aff8d10f-e00b-4dab-8466-52e15d2407a3?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by grahamsmith-okta — APPROVED** [review]

**Pull request merged**